tests/ui/parser/recover/recover-labeled-non-block-expr.stderr

```diff
@@ -54,7 +54,7 @@ help: consider enclosing expression in a block
    |
 LL ~     let _i = 'label: { match x {
 LL |         0 => 42,
- ...
+...
```

Comment:
Filed (almost trivial) https://github.com/rust-lang/rust/pull/126723 for these change, that I can rebase on top of once it gets merged.

Reply:
That's landed now.